src/test/java/utils/DriverSetUp.java

```diff
@@ -0,0 +1,41 @@
+package utils;
+
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.*;
+
+import java.io.*;
+import java.util.Properties;
+
+public class DriverSetUp {
+    protected static WebDriver driver;
+    private String filePath = "/Users/avoloshchuk/IdeaProjects/com.sharelane/src/test/resources/app.properties";
+    private Properties appProps = new Properties();
+
+    public void loadPropertiesFromFile() {
+        try (InputStream input = new FileInputStream(filePath)) {
+            appProps.load(input);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    @BeforeSuite
+    public void setChromeDriver() {
+        loadPropertiesFromFile();
+        System.setProperty(appProps.getProperty("chromeDriver"), appProps.getProperty("chromeDriverPath"));
+        driver = new ChromeDriver();
+    }
+
+    @BeforeClass
+    public void openBrowser() {
+        driver.get("https://www.sharelane.com/cgi-bin/main.py");
```

Comment:
there are 2 main parts: endpoint + somePage.
So endpoint could reused and is environment specific value - that's why it's better to place it to some env preperty or constant.
PageName could be changed in the future, the better option is to keep it in some constant too.

Reply:
Changed.
Path was divided into 2 parts: Endpoint+MAIN_PAGE
Endpoint was added to app.properties file.
MAIN_PAGE static final variable was created in the class.